### DISCUSSION TITLE
How to become an adopter of Theia 😄

### DISCUSSION THREAD

**Discussion by inlann** [eclipse-theia/theia#10657] [General]
Hello:

We are very grateful to the outstanding framework Theia 🥰 and we created an IDE named ZStudio base on Theia although we haven't officially released it yet 👻

We are also happy to contribute to Theia if possible and we'd love to become a contributor😄. We tried to make a PR here: https://github.com/eclipse-theia/theia/issues/10578.

We'd love to become an adopter too although our company is not well known at the moment 😂. Could you give us some information on how to become a Theia adopter? 

Really appreciate😊

**Comment by vince-fugnitto** [comment]
@inlann welcome to the community, glad to hear you were interested in becoming an adopter! :+1: 

There is no formal process for adopting Eclipse Theia, but you may be interested in the following:
- Adding your company logo to the list of adopters on the [homepage](https://theia-ide.org/) - (ex: https://github.com/eclipse-theia/theia-website/pull/246).
- Possibly attending the weekly [developer meetings](https://github.com/eclipse-theia/theia/wiki/Dev-Meetings).
- Contributing issues, reviews and pull-requests which you've already expressed desire to do :smiley: 

We look forward to hearing more from you!

cc @marcdumais-work @JonasHelming

**Reply by vhemery** [reply]
For the first point, also more information on https://ecdtools.eclipse.org/adopters/get-listed/

**Reply by inlann** [reply]
Hello @vhemery :

Thanks for response 😄 

> For the first point, also more information on https://ecdtools.eclipse.org/adopters/get-listed/

Does this mean that in addition to submitting a PR to [theia-website](https://github.com/eclipse-theia/theia-website), I also need to submit a PR to [eclipsefdn-project-adopters](https://github.com/EclipseFdn/eclipsefdn-project-adopters)?

**Reply by vhemery** [reply]
I'm afraid the two lists are independant for now... (I actually haven't modified theia-website for my company, maybe it would be worth taking information for theia-website from ecdtools)

**Reply by JonasHelming** [reply]
We plan to replace the list on the Theia website with the one on ECDT.eclipse.org (i.e. import it)

**Reply by JonasHelming** [reply]
For now, there are indeed two lists

**Reply by inlann** [reply]
Yes, they are indeed two lists for now. So that means I need to submit a PR to both of them at the moment?

**Reply by inlann** [reply]
I noticed that some companys on the theia-website list is not listed on the eclipsefdn-project-adopters list such as `huawei`

**Reply by JonasHelming** [reply]
Yes, please submit two PRs for now. We are in the process of consolidating this

**Reply by inlann** [reply]
Thanks so much 😊

**Comment by inlann** [comment]
I submitted a PR to theia-website #https://github.com/eclipse-theia/theia-website/pull/256 but I met some problems 😢. What should I do to fix all failed checks?

**Reply by vhemery** [reply]
Have you tried the following steps ?
https://www.eclipse.org/legal/ecafaq.php#collapseTen
If you have submitted with the wrong email, the simplest is probably to just close the PR and restart a new one with the correct mail...

**Reply by inlann** [reply]
Really appreciate!! I'll check these steps 👍🏼
